app/qml/FeatureForm.qml

```diff
@@ -441,16 +446,39 @@ Item {
         }
 
         Label {
-          id: constraintDescriptionLabel
+          id: fieldHelperText
+
+          property string constraintText: {
+            if ( ValueValidity === FormItem.ValueOutOfRange )
+              return qsTr( 'Number is outside of specified range' )
+            else if ( ValueValidity === FormItem.InvalidValue )
+              return qsTr( 'Value is not valid' )
+
+            if ( ConstraintDescription )
+              return ConstraintDescription
```

Comment:
logic should be in c++

Reply:
Will be in https://github.com/lutraconsulting/input/issues/1507